### PR TITLE
New reminder - closes Issue #7

### PR DESCRIPTION
**Pull request by sljohnson32**
@martensonbj @Tman22 

## Purpose

This PR introduces the ability for a user to add a new reminder to the page at route '/reminders/new'
Issue #7 

## Approach

This PR created a 'new' route and a button to 'new-reminder' component that renders a form.

### Test coverage 

We added tests for the reminder model and acceptance tests for the new template and new-reminder component.

### Follow-up tasks

Issue #8

### CODE REVIEW

File: app/routes/reminders/new.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import Ember from 'ember';
```
Comment (martensonbj) [inline app/routes/reminders/new.js:1]: If you aren't using this file let's delete it.

File: app/components/new-reminder.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+export default Ember.Component.extend({
+  store: Ember.inject.service(),
+
+  title: '',
+  date: '',
+  notes: '',
+
+  actions: {
+    addReminder() {
+      console.log(this.getProperties('date'));
```
Comment (martensonbj) [inline app/components/new-reminder.js]: Y'all know what the date is...console logs don't belong in master ;)

File: app/components/new-reminder.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+      const reminder = this.getProperties('title', 'date', 'notes');
+      this.get('store').createRecord('reminder', reminder).save().then(() => {
+        this.setProperties({
```
Comment (martensonbj) [inline app/components/new-reminder.js]: This can be one line

**Review by martensonbj — CHANGES_REQUESTED** [review]
Nice work guys. Couple small things

File: app/routes/reminders/new.js (hunk 1 above)
Comment (sljohnson32) [reply app/routes/reminders/new.js:1]: @martensonbj We are using this for the route that our new-reminder component lives on. If I comment out this code, our route breaks.  Is there a better way to do this?

@mlimberg

**Comment by sljohnson32** [comment]
@martensonbj @Tman22 I pushed up 2 of the 3 changes but left the 'new' route because I think we still need it.  Let us know if you don't think that's the case and we'll dig in deeper.

Thanks!

@mlimberg

**Comment by Tman22** [comment]
Looks good, nice work.

**Pull request merged**